hypothesis-python/tests/array_api/test_indices.py

```diff
@@ -72,7 +73,7 @@ def test_indices_replaces_whole_axis_slices_with_ellipsis(idx):
 
 
 @given(xps.indices((3, 3, 3, 3, 3)))
-def test_indices_effeciently_generate_indexers(_):
+def test_effeciently_generate_indexers(_):
```

Comment:
I just noticed the existing typo for `efficiently`.  Luckily `git grep` reveals it's the only such case 😅 

(and name this either "can efficiently generate" or "efficiently generates")